content/docs/insights/accounts.md

```diff
@@ -130,14 +130,49 @@ The AWS scanner for Pulumi Cloud requires access to the AWS account you want to
 
 ### Azure
 
-The Azure scanner for Pulumi Cloud requires access to your Azure account. This access can be granted by creating an ESC environment that, when opened, produces valid credentials to use the Pulumi Azure provider. Below are the steps to configure Azure credentials.
+The Azure scanner for Pulumi Cloud requires access to your Azure account. This access can be granted by creating an ESC environment that, when opened, produces valid credentials to use the Pulumi Azure provider.
+
+{{< notes type="info" >}}
+  We recommend using OpenID Connect (OIDC) for passwordless authentication with Azure. This method is more secure than using client secrets as it eliminates the need to store and manage long-lived credentials.
+{{< /notes >}}
+
+#### Option 1: OIDC authentication (recommended)
+
+1. Create a Microsoft Entra application and configure federated credentials:
+   * Follow the steps in [Configuring OpenID Connect for Azure](/docs/esc/environments/configuring-oidc/azure/)
+   * When configuring the federated credential:
+     * **Audience**: `azure:{your-org-name}`
+     * **Subject identifier**: `pulumi:environments:org:{your-org-name}:env:{project-name}/{environment-name}`
+   * Note the Application (client) ID, Directory (tenant) ID, and Subscription ID
+
+2. Use the following ESC configuration:
+
+```yaml
+values:
+  azure:
+    login:
+      fn::open::azure-login:
+        clientId: <YOUR_CLIENT_ID>
+        tenantId: <YOUR_TENANT_ID>
+        subscriptionId: <YOUR_SUBSCRIPTION_ID>
+        oidc: true
+  environmentVariables:
+    ARM_USE_OIDC: 'true'
+    ARM_CLIENT_ID: ${azure.login.clientId}
+    ARM_TENANT_ID: ${azure.login.tenantId}
+    ARM_OIDC_TOKEN: ${azure.login.oidc.token}
+    ARM_SUBSCRIPTION_ID: ${azure.login.subscriptionId}
+```
+
+#### Option 2: Client secret authentication
 
 1. Create a Service Principal in Azure, then generate the following values:
    * **clientId** (also called **appId** in the Azure UI)
    * **tenantId**
    * **subscriptionId**
    * **clientSecret** (also called **password** in the Azure UI)
-2. Use the following ESC configuration to provide the required credentials:
+
+2. Use the following ESC configuration:
```

Comment:
```suggestion
1. Use the following ESC configuration:
```

Ordered lists will autonumber. Using `1.` everywhere will make future diffs easier to read.

Reply:
Need to make sure this won't mess up the numbering further down, but looks like this is the end of the list.